sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -572,27 +570,14 @@ class EthService(
           blockContext.getEvmResult
         })
 
-        if(config!=null && config.getTracer!=null) {
-          // callTracer
-          if (config.getTracer == NativeTracers.CALL_TRACER.toString) {
-            val callTracerBlockView = new CallTracerBlockView(evmResults.toArray)
-            callTracerBlockView.getCallTracerTransactionViews
-          }
-          // 4byteTracer
-          else if (config.getTracer == NativeTracers.FOUR_BYTE_TRACER.toString) {
-            val fourByteTracerReturnList = new ListBuffer[mutable.Map[String, Integer]]
-            for (evmResult <- evmResults) {
-              if(evmResult!=null)
-                fourByteTracerReturnList += mapAsScalaMap(evmResult.fourByteTracerLogs)
-            }
-            fourByteTracerReturnList
-          }
-        }
-        // Struct/opcode logger
-        else {
-          val debugTraceBlockView = new DebugTraceBlockView(evmResults.toArray)
-          debugTraceBlockView.getDebugTraceTransactionViews
+        // return the list of tracer results from the evm
+        val tracerResultList = new ListBuffer[Any]
```

Comment:
Here too, would be a bit more clear to return a list of `JsonNode` instead of `Any`.

Reply:
Done